src/by/epam/ayem/module5/service/DragonsTreasureApp.java

```diff
@@ -42,21 +45,13 @@ public void run() {
 
             switch (choice) {
                 case 1:
```

Comment:
Magic number. What 1 means? Please use Enums or in worse case constants

Reply:
Sorry, I can't get it. Why is it bad to use integers for switch operator?